arches/app/views/controlled_lists.py

```diff
@@ -0,0 +1,313 @@
+from collections import defaultdict
+from datetime import datetime
+
+from django.core.exceptions import ValidationError
+from django.db import IntegrityError, transaction
+from django.db.models import Max
+from django.views.generic import View
+from django.utils.decorators import method_decorator
+from django.utils.translation import get_language, gettext as _
+
+from arches.app.models.models import (
+    ControlledList,
+    ControlledListItem,
+    ControlledListItemLabel,
+    Language,
+)
+from arches.app.utils.betterJSONSerializer import JSONDeserializer
+from arches.app.utils.decorators import group_required
+from arches.app.utils.response import JSONErrorResponse, JSONResponse
+
+
+def serialize(obj, depth_map=None):
```

Comment:
This probably opens up a larger conversation / can of worms but moving forward I'd like to move as much logic out of views as possible, and have them essentially just handle parsing args, calling other functions, and handling responses.

Reply:
I like it. So is the idea to break this up into three smaller functions and put them under /serializers?